src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -55,6 +55,7 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
       <a href="/login">Login</a>
     <% } %>
     <a href="/about.jsp">About</a>
+    <a href="/activityfeed">Activity Feed</a>
```

Comment:
You're going to want to add your Profile tab to all of these jsp files as well. Let me know if you are doing that in the next pull request instead because that is OK too

Reply:
I did it in the next pull request

Reply:
Yes, it will be done in the small PR